docs/databases/databases.html

```diff
@@ -102,29 +102,36 @@ <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
         <h1 class="heading-1">What are Databases?</h1>
         <p class="block-text">A database is a computerized system that is used to host a collection of information which is organized in a way that allows it to be easily accessed, managed and updated.</p>
         <p class="block-text">Databases exist everywhere in the real world. For example, your local supermarket will have a database that stores information regarding all its products. When the cashier scans an item at the checkout a request is made to their database using the unique barcode for that item. The price and name for that item are retrieved from the database, this information is then printed to your receipt.</p>
-        <p class="block-text">There are two main types of databases, SQL (or relational) and NoSQL (or non-relational) and Kitura has support for both types.</p>
+        <p class="block-text">There are two main types of databases, SQL (or relational) and NoSQL (or non-relational) and Kitura has support for both types. In this guide, we will describe SQL databases first and how Kitura provides support for them, then we will define what is meant by the term NoSQL database and list which NoSQL databases Kitura supports.</p>
```

Comment:
The first sentence here doesn't look right. Perhaps would read better as:

"There are two main types of databases, SQL (or relational) and NoSQL (or non-relational). Kitura has support for both types."

Reply:
👍